### DISCUSSION TITLE
Hacktoberfest 2022

### DISCUSSION THREAD

**Discussion by deeprobin** [dotnet/runtime#76515] [General] [answered]
Followup #43005.

Would it be possible to give the `hacktoberfest` topic to the runtime repository for this month?

Other repositories in the .NET Organization have done this as well (see `aspnetcore`, ...).

The question is not entirely altruistic either. The runtime is one of the main repositories where I contribute and I don't want to contribute to other repos just to get a stupid t-shirt.

/cc @danmoseley 
/cc @terrajobst

**Comment by stephentoub** [comment, marked as answer]
I added the tag.

**Reply by deeprobin** [reply]
Thank you!
